cmd/ipfs-cluster-service/daemon.go

```diff
@@ -36,8 +37,9 @@ import (
 	cli "github.com/urfave/cli"
 )
 
-func parseBootstraps(flagVal []string) (bootstraps []ma.Multiaddr) {
-	for _, a := range flagVal {
+func parseBootstraps(flagVal string) (bootstraps []ma.Multiaddr) {
+	addrs := strings.Split(flagVal, ",")
+	for _, a := range addrs {
```

Comment:
If the flag is empty, addrs will always be an slice of length 1, with an empty string. Thus you need to skip empty or check `flagVal` as more than 0 length.